stratum/hal/lib/phal/attribute_group.cc

```diff
@@ -965,10 +977,11 @@ ::util::Status AttributeGroupInternal::TraverseQuery(
         attribute_function) {
   auto reader_lock = AcquireReadable();
   absl::ReaderMutexLock lock(&registered_query_lock_);
-  auto query_info = gtl::FindOrNull(registered_queries_, query);
```

Comment:
What's the reason for not using `gtl::FindOrNull` here and further down?

If there is a compiler error it should be fixed (by me probably), instead of using workarounds.

Reply:
`gtl::FindOrNull` does not support `unique_ptr` at the moment. This workaround is thus fine.